packages/changeset-validations/package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "@ember-headless-form/changeset",
```

Comment:
does this package have a README with usage instructions?

Reply:
No, same as the main one! 😅

I plan to add that as part of the documentation efforts (#4)